index.bs

```diff
@@ -732,6 +748,21 @@ Given a [=request=] |request|:
 1. [=Queue a task=] to [=trigger attribution=] with |trigger|.
 1. Return <strong>blocked</strong>.
 
+<h3 dfn id="does-filter-data-match">Does filter data match</h3>
```

Comment:
Do you envision this and the other filtering related algorithms all going in the triggering algorithms section? Should we add a different section dedicated to filtering?

Reply:
Filtering only happens during triggering/attribution, so yes, I think they'd all go in the triggering section.